crates/db_schema/src/utils.rs

```diff
@@ -204,6 +206,20 @@ pub fn post_to_comment_sort_type(sort: SortType) -> CommentSortType {
   }
 }
 
+pub fn post_to_person_sort_type(sort: SortType) -> PersonSortType {
+  match sort {
+    SortType::Active | SortType::Hot | SortType::Controversial => PersonSortType::TopAll,
+    SortType::New | SortType::NewComments => PersonSortType::New,
+    SortType::MostComments => PersonSortType::MostComments,
+    SortType::Old => PersonSortType::Old,
+    SortType::TopDay => PersonSortType::TopDay,
+    SortType::TopAll =>  PersonSortType::TopAll,
+    SortType::TopWeek =>  PersonSortType::TopWeek,
+    SortType::TopYear =>  PersonSortType::TopYear,
+    SortType::TopMonth => PersonSortType::TopMonth,
+  }
+}
```

Comment:
Bit awkward that this is needed, but I guess there is no other way to make search all work.

Reply:
Do you think the same should be done with CommunitySortType or is it too much?

Reply:
It probably should, but you don't need to do that in this PR unless you want to. A lot of the Post sorts don't make sense for communities either.